src/main/webapp/stats.html

```diff
@@ -9,22 +9,35 @@
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
     integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
   <link href="https://fonts.googleapis.com/css?family=Mali&display=swap" rel="stylesheet">
+  <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+  <script async defer
+    src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDmsBaZDSm50H1fqB3PVmAr-BIwopAdsOg&callback=initMap"
+    type="text/javascript"></script>
+  <script type="text/javascript" src="./js/chart.js"></script>
+  <script src="./js/stats.js"></script>
 </head>
 
-<body>
+<body onload="buildUI()">
   <nav>
     <div class="brand">
       <h2> Play Even Harder </h2>
     </div>
     <ul id="navigation">
       <li><a href="#"></a></li>
     </ul>
+    <div class="search-container">
```

Comment:
I think only this div changes are necessary in this file. Everything else should be changed back.